pint_pandas/pint_array.py

```diff
@@ -864,15 +864,22 @@ def convert_values(param):
             # a TypeError should be raised
             res = op(lvalues, rvalues)
 
+            subdtype = self.dtype.subdtype
+            if "truediv" in op.__name__ and subdtype.kind == "i":
```

Comment:
In this case self can be:
```
<PintArray>
[1581.2004963851048, 2322.7768972177428, 2065.2759837278822,
 2108.0321074762774,  1922.714515192993]
Length: 5, dtype: pint[USD * percent][float64]
```
self.dtype is `pint[USD * percent][float64]` and self.dtype.subdtype is `'float64'`.  But the string `'float64'` does not have a `kind` attribute.